<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreetViewRatingAppClass</class>
 <widget class="QDialog" name="StreetViewRatingAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>494</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>779</width>
    <height>464</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>779</width>
    <height>494</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Adobe Gothic Std B</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Street View Rating App @ HPSCIL</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="loadDirBtn">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Open Dir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Save Scores</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="filepathLbl">
     <property name="minimumSize">
      <size>
       <width>731</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="imgLbl">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>351</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>16</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Rating</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="valueSlider">
       <property name="maximumSize">
        <size>
         <width>661</width>
         <height>21</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="valueBox">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="statusLbl">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="skipBtn">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Skip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextBtn">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Rating !</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="StreetViewRatingApp.qrc"/>
 </resources>
 <connections/>
</ui>
